File: app/src/main/java/co/deonna/solitare/Card.java
package co.deonna.solitare;

/**
 * Created by deonna on 6/4/17.
 */
public class Card {

    private final int value;
    private final String suit;
    private boolean faceUp;

    public Card(int value, String suit, boolean faceUp) {

        this.value = value;
        this.suit = suit;
        this.faceUp = faceUp;
    }

    public int getValue() {
        return value;
    }

    public String getSuit() {
        return suit;
    }

    public boolean isFaceUp() {
        return faceUp;
    }

    public void setFaceUp(boolean faceUp) {
        this.faceUp = faceUp;
    }
}

File: app/src/main/java/co/deonna/solitare/Card.kt
package co.deonna.solitare

val clubs = "Clubs"
val diamonds = "Diamonds"
val hearts = "Hearts"
val spades = "Spades"
val redCards = arrayOf(diamonds, hearts)
val blackCards = arrayOf(clubs, spades)

data class Card(val value: Int, val suit: String, var faceUp: Boolean = false)